argonaut/src/main/scala/org/http4s/argonaut/ArgonautInstances.scala

```diff
@@ -32,6 +32,15 @@ trait ArgonautInstances {
   def jsonEncoderWithPrinterOf[A](prettyParams: PrettyParams)
     (implicit encoder: EncodeJson[A]): EntityEncoder[A] =
     jsonEncoderWithPrettyParams(prettyParams).contramap[A](encoder.encode)
+
+  implicit val urlCodec: CodecJson[Uri] = CodecJson(
```

Comment:
It's not required, but it allows you to parse into `Uri` directly, instead of going via `String`.